web/src/views/repo/pipeline/PipelineDebug.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <template v-if="repoPermissions && repoPermissions.push">
+    <Panel>
+      <InputField :label="$t('repo.pipeline.debug.example_exec_title')">
+        <p class="text-sm text-wp-text-alt-100 mb-2">{{ $t('repo.pipeline.debug.example_exec_desc') }}</p>
+        <pre class="code-box">{{ cliExecWithMetadata }}</pre>
+      </InputField>
+      <div class="flex items-center space-x-4">
+        <Button :is-loading="isLoading" :text="$t('repo.pipeline.debug.download_metadata')" @click="downloadMetadata" />
+      </div>
+    </Panel>
+  </template>
+  <div v-else class="flex items-center justify-center h-full">
+    <div class="text-center p-8 bg-wp-control-error-100 rounded-lg shadow-lg">
+      <p class="text-2xl font-bold text-white">{{ $t('repo.pipeline.debug.no_permission') }}</p>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { inject, ref, type Ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+import Button from '~/components/atomic/Button.vue';
+import InputField from '~/components/form/InputField.vue';
+import Panel from '~/components/layout/Panel.vue';
+import useApiClient from '~/compositions/useApiClient';
+import useNotifications from '~/compositions/useNotifications';
+import type { Pipeline, Repo, RepoPermissions } from '~/lib/api/types';
+
+const { t } = useI18n();
+const apiClient = useApiClient();
+const notifications = useNotifications();
+
+const repo = inject<Ref<Repo>>('repo');
+const pipeline = inject<Ref<Pipeline>>('pipeline');
+const repoPermissions = inject<Ref<RepoPermissions>>('repo-permissions');
+
+const isLoading = ref(false);
+
+const downloadFileName = `${repo?.value.full_name.replaceAll('/', '-')}-pipeline-${pipeline?.value.number}-metadata.json`;
+const cliExecWithMetadata = `# woodpecker-cli exec --metadata-file ${downloadFileName}`;
```

Comment:
This needs to be a `computed` as repo / pipeline can change their values:

```suggestion
const metadataFileName = computed(() => `${repo?.value.full_name.replaceAll('/', '-')}-pipeline-${pipeline?.value.number}-metadata.json`;
const cliExecWithMetadata = computed(() => `# woodpecker-cli exec --metadata-file ${metadataFileName.value}`);
```